camel/agents/chat_agent.py

```diff
@@ -66,7 +66,7 @@
     from agentops import track_agent
 except ImportError:
 
-    def track_agent():
+    def track_agent(*args, **kwargs):
         def noop(f):
             return f
 
```

Comment:
![alt text](image.png)

We have define `track_agent` if it can not import track_agent. Need to move it to `track_agent`      file?